### PR TITLE
ADD PR template

### LINKED ISSUE #56: Issue template for PR
Since we are dealing with a lot of spreadsheets, and their conversion to Markdown files, it might be useful to have a PR template that points out some of these things:
- update acknowledgement: `data/acknowledgment.csv`
- update existing author's social media link: `data/affiliation_and_consent_for_the_brainhack_neuroview_preprint_raw.tsv`

thoughts?

### PR DESCRIPTION
**Pull request by htwangtw**
Closes #56 and replace #60

* add PR template
* description related to data files and relevant rules

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+This is a suggested pull request template for Brainhack Jupyer Book.
+It's designed to capture information we've found to be useful in reviewing pull requests.
+
+We do not accept edits on manuscript on GitHub.
+Please refer to the preprint: https://psyarxiv.com/rytjq/
+Instruction to annotate the preprint: https://help.osf.io/hc/en-us/articles/360019738554-Annotate-a-Preprint
+
+The pages related to authors and sponsors are built form spreadsheets under `data`.
+Please update the following information accordingly:
+Genral acknowledgements: `data/acknowledgments/acknowledgments.csv`
```
Comment (SamGuay) [inline .github/pull_request_template.md:11]:
```suggestion
General acknowledgements: `data/acknowledgments/acknowledgments.csv`
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+-->
+
+<!-- You can link a pull request to an issue to show that a fix is in progress.
+The issue will be automatically close when the pull request is merged.
+https://help.github.com/articles/closing-issues-using-keywords -->
+Closes # .
+
+<!-- Please give a brief overview of what has changed in the PR.
+If you're not sure what to write, consider it a note to the is to indicate
+what they should be looking for when they review the pull request. -->
```
Comment (SamGuay) [inline .github/pull_request_template.md:26]: I think there is a missing word in: 
> .... consider it a note to the `is` to indicate

**Review by SamGuay — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (htwangtw) [reply .github/pull_request_template.md]: Done!

**Review by SamGuay — APPROVED** [review]
All good now! :)

**Review by shawnrhoads — APPROVED** [review]
LGTM!

**Review by anibalsolon — APPROVED** [review]
Sorry for the delay here- Looks great! 🧸

If people keep skipping the commented content and submit manuscript edits, we can try later to upper-caps this part in the comment.

**Pull request merged**